content/docs/1.8.0/deploy/install/install-with-helm-controller.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Install with Helm Controller
+weight: 10
+---
+
+In this section, you will learn how to install Longhorn with the HelmChart controller built into RKE2 and K3S.
+
+### Prerequisites
+
+- Kubernetes cluster: Ensure that each node fulfills the [installation requirements](../#installation-requirements).  Cluster should be running RKE2 or K3S.
+
+> [This script](https://github.com/longhorn/longhorn/blob/v{{< current-version >}}/scripts/environment_check.sh) can be used to check the Longhorn environment for potential issues.
+
+### Installing Longhorn
+
+
+> **Note**:
+> * The initial settings for Longhorn can be [customized using Helm options or by editing the deployment configuration file.](../../../advanced-resources/deploy/customizing-default-settings/#using-helm)
+> * For Kubernetes < v1.25, if your cluster still enables Pod Security Policy admission controller, set the helm value `enablePSP` to `true` to install `longhorn-psp` PodSecurityPolicy resource which allows privileged Longhorn pods to start.
+
+
+1. Create a HelmChart yaml file similar to this:
+
+    ```yaml
+   apiVersion: helm.cattle.io/v1
+    kind: HelmChart
+    metadata:
+      annotations:
+        helmcharts.cattle.io/managed-by: helm-controller
+      finalizers:
+      - wrangler.cattle.io/on-helm-chart-remove
+      generation: 1
+      name: longhorn-install
+      namespace: default
+    spec:
+      version: v1.8.0
```

Comment:
Replace with a version variable. Same for other versions.

Reply:
Done, thanks.